integration-tests/tests/client/main.rs

```diff
@@ -0,0 +1,4 @@
+mod challenges;
+mod process_blocks;
+mod runtimes;
+mod sandbox;
```

Comment:
```suggestion
#[cfg(feature = "sandbox")]
mod sandbox;
```

so that we don't have to gate individual tests.

Reply:
Done